app/v1/controllers/RetrieveBalanceController.scala

```diff
@@ -22,19 +22,23 @@ import javax.inject.{Inject, Singleton}
 import play.api.libs.json.Json
 import play.api.mvc.{Action, AnyContent, ControllerComponents}
 import play.mvc.Http.MimeTypes
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.audit.http.connector.AuditResult
 import utils.Logging
 import v1.controllers.requestParsers.RetrieveBalanceRequestParser
 import v1.hateoas.HateoasFactory
+import v1.models.audit.{AuditDetail, AuditEvent, AuditResponse}
 import v1.models.errors._
 import v1.models.request.retrieveBalance.RetrieveBalanceRawRequest
 import v1.models.response.retrieveBalance.RetrieveBalanceHateoasData
-import v1.services.{EnrolmentsAuthService, MtdIdLookupService, RetrieveBalanceService}
+import v1.services.{AuditService, EnrolmentsAuthService, MtdIdLookupService, RetrieveBalanceService}
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class RetrieveBalanceController @Inject()(val authService: EnrolmentsAuthService,
                                           val lookupService: MtdIdLookupService,
+                                          val auditService: AuditService,
```

Comment:
All good, any reason why you add as **val**

Reply:
Not really, just wanted to keep it looking consistant as the other services were vals too. I can remove if needed.